Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1142,7 +1142,7 @@ return
 							Type = "number",
 						},
 					},
-					Notes = "Sets the block meta at the specified relative coords",
+					Notes = "Sets the block meta at the specified relative coords. Any call to SetBlockMeta will not generate a simulator update (water, lava, redstone), consider using SetBlock instead.",
```

Comment:
This is the API for cChunkDesc right? That's just a container for block data during World generation, no setters will generate any simulator updates.